python/RDS_SNAPSHOT_ENCRYPTED/RDS_SNAPSHOT_ENCRYPTED.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+"""""
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  RDS_SNAPSHOT_ENCRYPTED
+
+Description:
+  Check whether AWS RDS Snapshot is encrypted.
+
+Rationale:
+  Encrypting a  AWS RDS Snapshot with AWS Key Management Service(KMS) keys ensures protection.
+
+Indicative Severity:
+  Medium
+
+Trigger:
+  Configuration Change on AWS::RDS::DBSnapshot and AWS::RDS::DBClusterSnapshot
+
+Reports on:
+  AWS::RDS::DBSnapshot and AWS::RDS::DBClusterSnapshot
+
+Rule Parameters:
```

Comment:
Need to add a KmsArns parameter, similar to https://docs.aws.amazon.com/config/latest/developerguide/sagemaker-notebook-kms-configured.html

Reply:
This is Configuration change event on RDS Snapshot/Cluster. So We are not interested on encryption keys(KMS keys) used to encrypt, instead we are interested on  "encrypted" flag inside the configuration_item to decide COMPLIANT/NON_COMPLIANT.

Reply:
The point is to help customers to have the option to check addiitonally for a specific KMS key. The input parameter KmsArn is very relevant, check all the exisiting rules about encryption. You will need to iterate on the Gherkin.